### ISSUE TITLE
Missing static ODBC object file in installation script 

### ISSUE THREAD

**Issue by Superlokkus** [SOCI/soci#337]
When I built the current master branch with mingw on windows with only following cmake variables set in SOCI and WITH (rest default)

	SOCI
		SOCI_STATIC
		SOCI_ODBC
	WITH
		WITH_ODBC
		WITH_THREADS
		
libsoci_odbc_4_0.a is missing in the installation script thus is missing in the lib64 directory, altough it has been built.

Resulting install_manifest.txt:
```
C:/Program Files (x86)/SOCI/include/soci/backend-loader.h
C:/Program Files (x86)/SOCI/include/soci/bind-values.h
C:/Program Files (x86)/SOCI/include/soci/blob-exchange.h
C:/Program Files (x86)/SOCI/include/soci/blob.h
C:/Program Files (x86)/SOCI/include/soci/boost-fusion.h
C:/Program Files (x86)/SOCI/include/soci/boost-gregorian-date.h
C:/Program Files (x86)/SOCI/include/soci/boost-optional.h
C:/Program Files (x86)/SOCI/include/soci/boost-tuple.h
C:/Program Files (x86)/SOCI/include/soci/connection-parameters.h
C:/Program Files (x86)/SOCI/include/soci/connection-pool.h
C:/Program Files (x86)/SOCI/include/soci/error.h
C:/Program Files (x86)/SOCI/include/soci/exchange-traits.h
C:/Program Files (x86)/SOCI/include/soci/into-type.h
C:/Program Files (x86)/SOCI/include/soci/into.h
C:/Program Files (x86)/SOCI/include/soci/noreturn.h
C:/Program Files (x86)/SOCI/include/soci/once-temp-type.h
C:/Program Files (x86)/SOCI/include/soci/prepare-temp-type.h
C:/Program Files (x86)/SOCI/include/soci/procedure.h
C:/Program Files (x86)/SOCI/include/soci/query_transformation.h
C:/Program Files (x86)/SOCI/include/soci/ref-counted-prepare-info.h
C:/Program Files (x86)/SOCI/include/soci/ref-counted-statement.h
C:/Program Files (x86)/SOCI/include/soci/row-exchange.h
C:/Program Files (x86)/SOCI/include/soci/row.h
C:/Program Files (x86)/SOCI/include/soci/rowid-exchange.h
C:/Program Files (x86)/SOCI/include/soci/rowid.h
C:/Program Files (x86)/SOCI/include/soci/rowset.h
C:/Program Files (x86)/SOCI/include/soci/session.h
C:/Program Files (x86)/SOCI/include/soci/soci-backend.h
C:/Program Files (x86)/SOCI/include/soci/soci-config.h
C:/Program Files (x86)/SOCI/include/soci/soci-platform.h
C:/Program Files (x86)/SOCI/include/soci/soci-simple.h
C:/Program Files (x86)/SOCI/include/soci/soci.h
C:/Program Files (x86)/SOCI/include/soci/statement.h
C:/Program Files (x86)/SOCI/include/soci/transaction.h
C:/Program Files (x86)/SOCI/include/soci/type-conversion-traits.h
C:/Program Files (x86)/SOCI/include/soci/type-conversion.h
C:/Program Files (x86)/SOCI/include/soci/type-holder.h
C:/Program Files (x86)/SOCI/include/soci/type-ptr.h
C:/Program Files (x86)/SOCI/include/soci/unsigned-types.h
C:/Program Files (x86)/SOCI/include/soci/use-type.h
C:/Program Files (x86)/SOCI/include/soci/use.h
C:/Program Files (x86)/SOCI/include/soci/values-exchange.h
C:/Program Files (x86)/SOCI/include/soci/values.h
C:/Program Files (x86)/SOCI/include/soci/version.h
C:/Program Files (x86)/SOCI/lib64/libsoci_core_4_0.a
C:/Program Files (x86)/SOCI/include/soci/odbc/soci-odbc.h
```

**Comment by mloskot** [comment]
Would you be able to provide a patch for that issue?

**Comment by Superlokkus** [comment]
I will try, but please be patient, I only get to work on it about once per two weeks until late July.

**Comment by mloskot** [comment]
Take your time. Thanks in advance!

**Comment by snikulov** [comment]
@mloskot @Superlokkus possible fixed with #403

**Comment by mloskot** [comment]
We confirmed it's been fixed.

@Superlokkus Please, reopen if you still experience this issue.
